content/docs/user-guide/setup-google-drive-remote.md

```diff
@@ -3,7 +3,13 @@
 Follow this guide to setup Google Drive as your <abbr>DVC project</abbr>'s
 [remote storage](/doc/command-reference/remote).
 
-## Configure a Google Cloud project
+## Configure a Google Cloud project (highly recommended)
+
+Optionally, follow this guide to setup your own Google Cloud project and
+[OAuth](https://developers.google.com/identity/protocols/OAuth2) credentials.
```

Comment:
OK. I'm not seeing how this is optional. You need a GC project to use either client creds (id/secret) or a service account. How can this be skipped @maxhora ? I.e. how do you use the default DVC project? Thanks

Reply:
@jorgeorpinel the change to use client id and secret from the default DVC project was made in the last my commit there https://github.com/iterative/dvc/commit/8a4827017ea171ef0dab49cb79785c997075bab6#diff-21bfd10d01c6715fc8634644730fa720R102
If user haven't specified client_id and client_secret in DVC config the default pair will be used by DVC (user will not need to create their own GCP project in this case).

Reply:
Ping me on Slack, if it's still not clear, @jorgeorpinel

Reply:
Got it. It's just not clear that you don't have to provide anything other than the gdrive URL, and then the first `dvc push` will give you a link to authorize the DVC app:

![image](https://user-images.githubusercontent.com/1477535/77976738-fea60c80-72ba-11ea-8d67-77133f02cb6f.png)

But I got it now I think. I'll clarify all this, thanks.

Reply:
Yes! Google drive is in fact the most simple of all remotes to start - and that's what we should clearly communicate. With GD you can start with just an URL. Then if you hit limits (you can check logs and see the message), you should create your own project. Service account and stuff - advanced features.

Also, might potentially check rclone's docs - how they document their shared credentials and how they motivate user on creating their own projects.

Reply:
OK I've mostly addressed this except didn't mention this progression about first just use URL, if you hit limits create a GC project, etc. Didn't check rclone docs yet either but will do ⏳

Reply:
I think this is addressed now. I did also check out https://rclone.org/drive/ but didn't find much inspiration anyway.